### PR TITLE
Make cherry pick commit color customisable

### PR DESCRIPTION
**Pull request by samburville**
Two new settings in the config, which allow the cherry picked
foreground and background to be custom colors.

Issue #856

### CODE REVIEW

File: pkg/theme/theme.go [hunk 1]
```diff
@@ -44,6 +47,11 @@ func UpdateTheme(themeConfig config.ThemeConfig) {
 	InactiveBorderColor = GetGocuiStyle(themeConfig.InactiveBorderColor)
 	SelectedLineBgColor = GetTextStyle(themeConfig.SelectedLineBgColor, true)
 	SelectedRangeBgColor = GetTextStyle(themeConfig.SelectedRangeBgColor, true)
+	
+	var cherryPickedCommitBgTextStyle =	GetTextStyle(themeConfig.CherryPickedCommitBgColor, true)
```
Comment (jesseduffield) [inline pkg/theme/theme.go:51]:
```suggestion
	cherryPickedCommitBgTextStyle :=	GetTextStyle(themeConfig.CherryPickedCommitBgColor, true)
```

File: pkg/theme/theme.go [hunk 2]
```diff
@@ -44,6 +47,11 @@ func UpdateTheme(themeConfig config.ThemeConfig) {
...
+	var cherryPickedCommitFgTextStyle =	GetTextStyle(themeConfig.CherryPickedCommitFgColor, false)
```
Comment (jesseduffield) [inline pkg/theme/theme.go:52]: small convention thing
```suggestion
	cherryPickedCommitFgTextStyle :=	GetTextStyle(themeConfig.CherryPickedCommitFgColor, false)
```

**Review by jesseduffield — COMMENTED** [review]
nice work! Just a couple suggestions

**Comment by samburville** [comment]
Ah thanks, have done as suggested

File: pkg/theme/theme.go [hunk 3]
```diff
@@ -44,6 +47,11 @@ func UpdateTheme(themeConfig config.ThemeConfig) {
 	InactiveBorderColor = GetGocuiStyle(themeConfig.InactiveBorderColor)
 	SelectedLineBgColor = GetTextStyle(themeConfig.SelectedLineBgColor, true)
 	SelectedRangeBgColor = GetTextStyle(themeConfig.SelectedRangeBgColor, true)
+	
+	cherryPickedCommitBgTextStyle :=	GetTextStyle(themeConfig.CherryPickedCommitBgColor, true)
```
Comment (jesseduffield) [inline pkg/theme/theme.go:51]:
```suggestion
	cherryPickedCommitBgTextStyle := GetTextStyle(themeConfig.CherryPickedCommitBgColor, true)
```

File: pkg/theme/theme.go [hunk 4]
```diff
@@ -44,6 +47,11 @@ func UpdateTheme(themeConfig config.ThemeConfig) {
 	InactiveBorderColor = GetGocuiStyle(themeConfig.InactiveBorderColor)
 	SelectedLineBgColor = GetTextStyle(themeConfig.SelectedLineBgColor, true)
 	SelectedRangeBgColor = GetTextStyle(themeConfig.SelectedRangeBgColor, true)
+	
+	cherryPickedCommitBgTextStyle := GetTextStyle(themeConfig.CherryPickedCommitBgColor, true)
+	cherryPickedCommitFgTextStyle :=	GetTextStyle(themeConfig.CherryPickedCommitFgColor, false)
```
Comment (jesseduffield) [inline pkg/theme/theme.go:52]:
```suggestion
	cherryPickedCommitFgTextStyle := GetTextStyle(themeConfig.CherryPickedCommitFgColor, false)
```

**Comment by jesseduffield** [comment]
will merge after tests pass :)

**Comment by jesseduffield** [comment]
Looks like it needs some text formatting. You can use `gofmt` to format the code

**Comment by samburville** [comment]
Hoping it's all fixed - just needs a workflow run

**Comment by jesseduffield** [comment]
Nice one @samburville !

**Pull request merged**